README.md

```diff
@@ -143,6 +149,9 @@ By Tommy Buonomo ([Frenchie Games](https://frenchiegames.app/#/))
 <!--lint ignore double-link-->
 - Dino Run - [Android](https://play.google.com/store/apps/details?id=com.nobs.dino_run) - Dino Run is casual 2D infinite side scroller game, where you play as a cute little dino. All you have to do is avoid getting hit by enemies too many times. The longer you can do that, higher your score goes. By [DevKage](https://www.youtube.com/devkage)
 
+<!--lint ignore double-link-->
+- Spacescape - [Android](https://play.google.com/store/apps/details?id=com.nobs.spacescape) - Spacescape is a simple and minimalistic 2D space shooter game made using the awesome Flame engine. Just shoot all the enemies coming your way to achieve the highest score. By [DevKage](https://www.youtube.com/devkage)
```

Comment:
I think we can combine these entries? the `App Releases` section was to allow non-open-source games to appear as well, if your game is open source it should be on the feature open source list and you can add both links

Reply:
Sorry, but I didn't understand what exactly you want me to do. 😅 
- Should I completely remove `Dino Run` and `Spacescape` from `App Releases`? 
- Or should I merge them into single entry? If so, should it be in any specific format?
- Or should I add Play Store links to corresponding entries under `Open Source`?

Reply:
sorry if it wasn't clear! I meant:

* you can definitely have separated entries for each game
* we should have only one entry on Open Source (if your game is open source) OR an entry on the non-open-source section (else). not both
* you can have github, store link, or any other links on the Open Source section

At least that is what I remembered we discussed about the non-open-source section years ago haha.
But I leave this for @erickzanardo and @impulse to confirm or deny.

Reply:
Okay, I think I get it now. I'll wait for others to reply. Maybe the `CONTRIBUTING.md` also needs some rework then.

Reply:
Removed `Dino Run` from `App Releases` section and added play store links under `Open Source` section.